src/gui/tray/NotificationHandler.h

```diff
@@ -26,6 +26,7 @@ private slots:
     void slotNotificationsReceived(const QJsonDocument &json, int statusCode);
     void slotEtagResponseHeaderReceived(const QByteArray &value, int statusCode);
     void slotIconDownloaded(QByteArray iconData);
+    void slotDesktopNotificationsAllowedReceived(const bool isAllowed);
```

Comment:
@camilasan Maybe, we need to change this slot's name and the corresponding signal's name to something easier to understand? Maybe, something like this: `slotDesktopNotificationsAllowedReceived` > `slotAllowDesktopNotificationsChanged`? and `DesktopNotificationsAllowedReceived` > `AllowDesktopNotificationsChanged`?